<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFormCreatorWidget</class>
 <widget class="QDialog" name="CFormCreatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="HorizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="LayoutName">
        <item>
         <widget class="QLabel" name="LabalName">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LineEditName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="LayoutValue">
        <item>
         <widget class="QLabel" name="LabelValue">
          <property name="text">
           <string>Value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LineEditValue"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
